### DISCUSSION TITLE
How to download auido from ui.audio?

### DISCUSSION THREAD

**Discussion by KimigaiiWuyi** [zauberzeug/nicegui#2456] [Q&A] [answered]
### Question

Maybe I'm missing something, but I really don't see it described in the documentation or issuse and discussions. 
Using the download button in the ui.audio menu and clicking on download, you'll get a grayed out download task and the download will be interrupted immediately.
Am I wrong about something?

**Comment by falkoschindler** [comment, marked as answer]
Hi @KimigaiiWuyi,

Did you check with the latest NiceGUI version [1.4.12](https://github.com/zauberzeug/nicegui/releases/tag/v1.4.12)? In PR https://github.com/zauberzeug/nicegui/pull/2250 we improved the way media files are served. This should fix your problem.

**Reply by KimigaiiWuyi** [reply]
I updated to the latest version and now everything works fine, thank you very much!
